admin_guide/runtime_defense/runtime_defense_hosts.adoc

```diff
@@ -41,19 +41,32 @@ NOTE: The *Prevent* action for detection of file system events requires a Linux
 
 - *Crypto miners* -- Apply specific techniques for detection of crypto miners, alert on file creation, and alert or prevent their execution.
 
-- *Non-packaged binaries created or run by service* -- Detect binaries created by a service without a package manager. alert on file creation, and alert or prevent their execution.
-
-- *Non-packaged binaries created or run by user* -- Detect binaries created by a user without a package manager. alert on file creation, and alert or prevent their execution. Alert on file creation, and alert or prevent their execution.
-
-- *Processes running from temporary storage* -- Detect processes running from temporary storage (unexpected behavior for legitimate processes). Alert/prevent on file creation or execution.
-
-- *Webshell attacks* -- Detect abuse of web servers vulnerabilities to create a webshell. Alert on webshell creation and and alert or prevent execution of linux command line tools from web servers.
+- *Non-packaged binaries created or run by service* -- Detect binaries created by a service without a package manager.
+Alert on file creation, and alert or prevent their execution.
++
+NOTE: Defender must be running when a file is written to detect its source. 
++
+NOTE: Currently supported only apt, yum, and dnf.
```

Comment:
@tomspivak This bullet is about non-packaged binaries. However, this note talks about supported package managers. Does this note need to be put somewhere else in this article? Or is there something I'm not understanding?

Reply:
The location here and below are correct. This feature is about detecting binaries that have been deployed without a package manager, but we depend on the package manager used on the host to be able to detect that, and as such we can only perform the detection if the specified package managers are used.

Reply:
@tomspivak Got it, thanks. I understand now. I'll update your note with this additional context, and then merge your PR.